README.md

```diff
@@ -21,124 +21,132 @@
 
-```Shell
-conda install orange3-<addon name>
-```
+Check out our widget development [![docs](https://readthedocs.org/projects/orange-widget-base/badge/?version=latest)](https://orange-widget-base.readthedocs.io/en/latest/?badge=latest) for a comprehensive guide on writing Orange widgets.
 
-See specific add-on repositories for details.
+If you're looking for a good starting point, check out our [![good first issues](https://img.shields.io/github/issues/biolab/orange3/good%20first%20issue?label=good%20first%20issues)](https://github.com/biolab/orange3/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).
 
-### Installing with pip
 
-To install Orange with pip, run the following.
+### The Orange Ecosystem
 
-```Shell
-# Install some build requirements via your system's package manager
-sudo apt install virtualenv build-essential python3-dev
+The development of Orange is primarily split into three repositories:
 
-# Create a separate Python environment for Orange and its dependencies ...
-virtualenv --python=python3 --system-site-packages orange3venv
-# ... and make it the active one
-source orange3venv/bin/activate
+[biolab/orange-canvas-core](https://www.github.com/biolab/orange-canvas-core) implements canvas elements,  
+[biolab/orange-widget-base](https://www.github.com/biolab/orange-widget-base) implements a widget window's interface elements,  
+[biolab/orange3](https://www.github.com/biolab/orange3) brings it all together and implements the base data mining toolbox.	
 
-# Install Orange
-pip install orange3
-```
+Additionally, add-ons implement additional widgets for more specific use cases. [Anyone can write an add-on.](https://github.com/biolab/orange3-example-addon) These are our first-party add-ons:
 
-### Installing with winget (Windows only)
+- [biolab/orange3-text](https://www.github.com/biolab/orange3-text)
+- [biolab/orange3-bioinformatics](https://www.github.com/biolab/orange3-bioinformatics)
+- [biolab/orange3-timeseries](https://www.github.com/biolab/orange3-timeseries)    
+- [biolab/orange3-single-cell](https://www.github.com/biolab/orange3-single-cell)    
+- [biolab/orange3-imageanalytics](https://www.github.com/biolab/orange3-imageanalytics)    
+- [biolab/orange3-educational](https://www.github.com/biolab/orange3-educational)    
+- [biolab/orange3-geo](https://www.github.com/biolab/orange3-geo)    
+- [biolab/orange3-associate](https://www.github.com/biolab/orange3-associate)    
+- [biolab/orange3-network](https://www.github.com/biolab/orange3-network)
 
-To install Orange with [winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/), run:
+### Setting up for development of core widgets
+
+Fork the repository by pressing the fork button in top-right corner of this page. Then execute the following lines (copy them one by one!):
```

Comment:
Why do these need to be copied one by one? Can you append ` &&` to all lines up to the penultimate, and remove this?

Reply:
If you copy them all at once, the second line is used as the answer to Y/n prompt of the first.

Lines include YOUR-USERNAME, so they're not very copiable. Even more so in the next section.

Reply:
Might I suggest this?
```bash
# Add conda-forge to your channels for access to the latest release
conda config --add channels conda-forge

# Create and activate an environment for orange
yes | conda create python=3 --name orange3
conda activate orange3

# Install orange
conda install orange3
```
I can't think of a single time I followed instructions telling me to copy line by line, especially if presented with a copiable code block.

I don't think the `yes` would be too confusing, it's probably the simplest bash command out there.